.github/renovate.json

```diff
@@ -21,7 +20,7 @@
     },
     {
       "enabled": true,
-      "groupName": "Block Protocol",
+      "groupName": "Block Protocol npm packages",
```

Comment:
Not worth investigating now, but having this automated would be nice at some point.

Reply:
I suspect it's doable. Renovate is **so** configurable. But it also **_so_** configurable I don't have the patience to wade through its voluminous docs right now. 😅